x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/action/TransportOpenJobAction.java

```diff
@@ -574,6 +592,35 @@ public void onTimeout(TimeValue timeout) {
         });
     }
 
+    private void clearJobFinishedTime(String jobId, ActionListener<OpenJobAction.Response> listener) {
+        clusterService.submitStateUpdateTask("clearing_job_finish_time [" + jobId + "]", new ClusterStateUpdateTask() {
```

Comment:
nit: it seems that in most places we use hyphens to name the cluster state update tasks. It would be more consistent to rename this to `clear-finished-time-for-job-{job_id}`.